### PR TITLE
changes to docs to use kathmandu for contracts for live code

### LINKED ISSUE #1875: Convert Taquito Docs live code examples to work with Kathmandunet
Each live code example in the Taquito docs should be changed to a contract that works on kathmandunet.

Follow the procedure to create the new contracts and then edit the docs code to set the new addresses.

use:  https://ecadlabs.ontestpad.com/script/316#//

then after done check it using:  https://ecadlabs.ontestpad.com/script/317#//

### PR DESCRIPTION
**Pull request by michaelkernaghan**
closes #1875 

update the live code examples in Taquito docs to use kathmandunet

also update the code example in ~/taquito/example

### CODE REVIEW

File: docs/estimate.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ Tezos.estimate
 
 ```js live noInline wallet
 // import { TezosToolkit } from '@taquito/taquito';
-// const Tezos = new TezosToolkit('https://jakartanet.api.tez.ie');
+// const Tezos = new TezosToolkit('https://kathmandunet.api.tez.ie');
```
Comment (roxaneletourneau) [inline docs/estimate.md:205]: `https://kathmandunet.api.tez.ie` is not a valid url

https://kathmandunet.ecadinfra.com

File: example/example-node.ts [hunk 2]
```diff
@@ -2,15 +2,15 @@ import { TezosToolkit } from '../packages/taquito/src/taquito';
...
+const provider = 'https://kathmandunet.ecadinfra.com/'
 const client = new RpcClient(provider);
 
 async function example() {
   try {
     const tezos = new TezosToolkit(provider);
 
     console.log('Getting storage...');
-    await tezos.contract.at('KT1AbzoXYgGXjCD3Msi3spuqa5r5MP3rkvM9').then(async contract => { // knownBigMapContract used in integration tests
+    await tezos.contract.at('KT1VBn1rTZ4FLz43hC9xugn56K9mSM2DQQXJ').then(async contract => { // knownBigMapContract used in integration tests
```
Comment (roxaneletourneau) [inline example/example-node.ts:13]: Not the right contract

File: example/example-node.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ async function example() {
     });
 
     console.log('Getting big map key...');
-    await tezos.contract.at('KT1AbzoXYgGXjCD3Msi3spuqa5r5MP3rkvM9').then(async contract => {
+    await tezos.contract.at('KT1VBn1rTZ4FLz43hC9xugn56K9mSM2DQQXJ').then(async contract => {
```
Comment (roxaneletourneau) [inline example/example-node.ts:24]: Not the right contract

File: example/example-sequential-bigmap-local-vs-rpc-pack.ts [hunk 4]
```diff
@@ -10,7 +10,7 @@ async function example() {
     try {
         const tezos = new TezosToolkit(rpc);
 
-        const contractAddress = 'KT1Hrp5i3P5BUATvkP7scdqY5PHBmYyz3CNA';
+        const contractAddress = 'KT1VBn1rTZ4FLz43hC9xugn56K9mSM2DQQXJ';
```
Comment (roxaneletourneau) [inline example/example-sequential-bigmap-local-vs-rpc-pack.ts:13]: Not the right contract, should be `KT1VqHMv2j61NSNKBPumRNjh8dDksTmnbGjs` I think

File: integration-tests/config.ts [hunk 5]
```diff
@@ -74,7 +74,7 @@ const kathmandunetEphemeral = {
   protocol: Protocols.PtKathman,
   signerConfig: {
     type: SignerType.EPHEMERAL_KEY as SignerType.EPHEMERAL_KEY,
-    keyUrl: 'http://key-gen-1.i.tez.ie:3000/kathmandunet',
+    keyUrl: 'https://api.tez.ie/keys/jakartanet',
```
Comment (roxaneletourneau) [inline integration-tests/config.ts:77]: kathmandunet, not jakartanet

File: package.json [hunk 6]
```diff
@@ -95,7 +95,7 @@
     "ts-node": "^10.4.0",
     "ts-toolbelt": "^9.6.0",
     "typedoc": "^0.20.36",
-    "typescript": "~4.1.5"
+    "typescript": "4.2"
```
Comment (roxaneletourneau) [inline package.json:98]: We should keep the same version as in the other packages: `~4.1.5`

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]

File: docs/estimate.md (hunk 1 above)
Comment (michaelkernaghan) [reply docs/estimate.md:205]: Fixed. also fixed another case in making_transfer.md

File: example/example-node.ts (hunk 3 above)
Comment (michaelkernaghan) [reply example/example-node.ts:24]: changed to KT1AwUJp6ozYtzhpf5wVXZPQSFxb64JFcVvi

Comment (michaelkernaghan) [reply example/example-node.ts:24]: Verified the example works with new contract

File: example/example-sequential-bigmap-local-vs-rpc-pack.ts (hunk 4 above)
Comment (michaelkernaghan) [reply example/example-sequential-bigmap-local-vs-rpc-pack.ts:13]: Yes, that is the correct replacement. Replaced and verified it works.

File: integration-tests/config.ts (hunk 5 above)
Comment (michaelkernaghan) [reply integration-tests/config.ts:77]: oops!!   thanks, nice catch :)

Fixed.

File: package.json (hunk 6 above)
Comment (michaelkernaghan) [reply package.json]: fixed.

**Review by roxaneletourneau — APPROVED** [review]
Good to go when the pipeline finish, please use "Squash and merge" :)

**Pull request merged**